app/views/subscription_mailer/confirmation.text.haml

```diff
@@ -1,13 +1,13 @@
 \# #{t('app.title')}
-\# #{@subscription_reference}
+\## #{t('subscriptions.email.confirmation.subject', reference: @subscription_reference)}
```

Comment:
Can we use `@subscription.reference` for this and save ourselves a level of indirection?